packages/features/bookings/components/TeamFilter.tsx

```diff
@@ -38,7 +38,7 @@ export const TeamsMemberFilter = () => {
           checked={!query.teamIds && !query.userIds?.includes(session.data?.user.id || 0)}
           onChange={() => {
             setDropdownTitle(t("all_bookings_filter_label"));
-            removeByKey("teamIds"); // Always clear on toggle  or not toggle (seems weird but when you know the behviour it works well )
```

Comment:
@sean-brydon I saw that you made this comment. Are you ok with this approach?

Reply:
Yeah works for me :)